Exercise-2.R

```diff
@@ -56,15 +60,21 @@ by_country
 
 # What happened inside this pipeline? --> (comment your answer here)
 
+#Data was grouped by the type of consent laws present in that country, then summarized by mean and standard deviation, then N/As were removed. The data was then ungrouped.
+
 # Now for the Cleveland dot plot:
 ggplot(data = by_country,
        mapping = aes(x = donors_mean, y = reorder(country, donors_mean),
                      color = consent_law)) + 
   geom_point(size=3) +
   labs(x = "Donor Procurement Rate",
        y = "", color = "Consent Law") +
-  theme(legend.position="top")
-
+  theme(legend.position="top")+
+  facet_wrap(~consent_law) +
+  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(),
+        panel.background = element_blank(), axis.line = element_line(colour = "black")) +
+  ggtitle("Effect of Consent Law Status on Donor Procurement") + theme(plot.title = element_text(hjust=1))
+  
 # Try adding a facet_wrap() by consent law to the plot above. Facet_wrap has additional arguments that you could explore, including scales =, and ncol=. Again, Google is your friend here.
 
 # Finally, add a title and remove gridlines. Once you are happy with your final Cleveland dot plot, save it.
```

Comment:
I think you may have had Plot 1 in your viewing pane when you executed this command.